bigtable/client/row_reader.cc

```diff
@@ -81,6 +81,46 @@ RowReader::RowReader(
       raise_on_error_(raise_on_error),
       error_retrieved_(raise_on_error) {}
 
+RowReader::RowReader(
+    std::shared_ptr<DataClient> client, std::string app_profile_id,
```

Comment:
In other places `app_profile_id` appears after the table, please use the same order everywhere to avoid mistakes, or consider strong types, or the struct wrapping `table_name, app_profile_id` idea mentioned above.